src/K4os.Compression.LZ4/LZ4Codec.cs

```diff
@@ -112,6 +112,25 @@ public static bool Enforce32
 			
 		return decoded <= 0 ? -1 : decoded;
 	}
+	
+	/// <summary>Decompresses data from given buffer, stopping at <paramref name="targetLength"/>.</summary>
+	/// <param name="source">Input buffer.</param>
+	/// <param name="sourceLength">Input buffer length.</param>
+	/// <param name="target">Output buffer.</param>
+	/// <param name="targetLength">Output buffer length. Decoding stops at this amount.</param>
+	/// <returns>Number of bytes written, or negative value if output buffer is too small.</returns>
+	public static unsafe int DecodePartial(
```

Comment:
A little bit of bike shedding and I know why this order happens, but I would rather have `PartialDecode` as it is a properly English sounding term.

Reply:
I don't mind whichever way; I'll flip it around.

Reply:
Done in c11d077daae087c28ab37730cb2af960c52e7340